### PR TITLE
Add script, which can be used for JS and Ruby projects

### PR DESCRIPTION
**Pull request by akulakovaa**

### CODE REVIEW

File: bin/bump.sh [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+    without changing versioning, and rewrite only commit hash inside '.tags' file.
+
+  Options:
+    -j, --js                    JavaScript repo.
+    -r, --ruby                  Ruby repo."
+
+kite_func() {
+    git config --global user.name "Kite Bot"
+    git config --global user.email "kite-bot@heliostech.fr"
+    #git remote add authenticated-origin https://github.com/Kohelbekker/barong
```
Comment (mnaichuk) [inline bin/bump.sh]: remove this line please.

**Review by ec — CHANGES_REQUESTED** [review]
@Anna-Kulakova https://ci.microkube.com/rubykube/barong/1008
This error means you are not rebased on the latest master.
Please rebase PR on master and drone will get up)

**Review by ec — APPROVED** [review]

**Comment by ec** [comment]
@Anna-Kulakova are guys still testing some stuff?
Can i freeze PR?

**Comment by ec** [comment]
let me know if you need to reopen it

**Pull request closed without merging**